internal/handlers/users_v1_handler.go

```diff
@@ -1,18 +1,86 @@
 package handlers
 
 import (
+	"context"
+	"encoding/json"
+	"io"
 	"net/http"
 
 	"github.com/redhatinsights/mbop/internal/config"
+	"github.com/redhatinsights/mbop/internal/service/ocm"
+
+	"github.com/redhatinsights/mbop/internal/models"
 )
 
 func UsersV1Handler(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+
 	switch config.Get().UsersModule {
 	case awsModule:
```

Comment:
We'll actually want a new constant defined here for `const amsModule = "ams"` vs `aws`?